nncf/compression_method_api.py

```diff
@@ -52,21 +52,21 @@ class PTCompressionLoss(nn.Module, CompressionLoss):
     and fully-connected layers to construct the loss function.
     """
 
-    def calculate(self) -> torch.Tensor:
+    def calculate(self, input_=None, target=None) -> torch.Tensor:
```

Comment:
@alexsu52 
@vshampor 
KDLossCalculator requires API changes because we need to get arguments (input_  and target) inside KDLossCalculator module to calculate loss. So should this changes need to be done for  PyTorch Compression API only or for Unified API too? In other words: will this changes affect TF Loss calculation policies?

Reply:
I suppose that since knowledge distillation loss fundamentally requires some sort of inputs and outputs, then the change must be done on the unified API level.

Reply:
+ @daniil-lyakhov

Reply:
NNCF TF compression loss have completely different api and do all work inside `call` function.  I think this doesn't affect TF and will simply unify in future. CC: @alexsu52

Reply:
NNCF TF compression loss have completely different api and do all work inside `call` function.  I think this doesn't affect TF and will simply unify in future. CC: @alexsu52

Reply:
created 58573 task to explore TF Part